config.rb

```diff
@@ -32,7 +32,7 @@ def render_markdown(text)
 
   def get_author(article_author)
     author = data.authors.detect { |author| author["name"].downcase == article_author.downcase }
-    author.nil? ? raise('No author found') : author
+    article_author.to_s.strip == '' ? raise('No author found') : author
```

Comment:
Is it so bad if the author doesn't exist?  Could just return nil and guard around it where you try to use it?